### PR TITLE
Update welcome.md

### PR DESCRIPTION
**Pull request by QuestionableTangerine**
Added FAQ from https://www.notion.so/BobaBoard-s-Welcome-Packet-0021b3cf252a4152ab8835f43ddebfee (which was in turn drawn partially from my original writing and partially from the discord FAQ)

### CODE REVIEW

File: docs/users/welcome.md [hunk 1]
```diff
@@ -22,3 +22,35 @@ What to do (and not to do) in the community.
...
+
+*a.k.a. answer to older threads that haven't seen activity in a while
+
+### I really need a post deleted!
+
+Just write under the post that you'd like for that to happen, and I'll do so after confirmation. If I don't notice the request, please bump it up in !bobaland, which I check most often. 
+
+If you see a deletion request in the wild, feel free to *privately* bring it up to me on Discord. You don't need to tell me if it's yours 😉
+
+### Is BobaBoard a [Insert Social Network] Clone?
```
Comment (essential-randomness) [inline docs/users/welcome.md:40]: I would rephrase this in a way that helps clarify expectations of what we have now as software vs what people can expect to see going forward. This FAQ makes sense in the abstract sense, but I'd keep the welcome guide more tightly focused on the practical. We can always link out as needed.

File: docs/users/welcome.md [hunk 2]
```diff
@@ -22,3 +22,35 @@ What to do (and not to do) in the community.
...
+
+BobaBoard isn't an anything clone. Even beyond the concept of anonymity-first, I strive to rethink our assumptions on the shape of online fandom spaces.
+
+Inspiration for the platform will come from many different sources: my Peak Fandom Fun was on Tumblr; I often come back to Amino Apps and Discord Servers as tools for the creation of smaller, "independent" hubs of fandom activity; Reddit and Quora offer insights on less-identity-centered spaces of discussion; And (I swear I'm not making this up) I've even considered lifting features from LinkedIn.
+
+There are many ways BobaBoard could evolve as it grows, yet it's important to not be prescriptive when it comes to product development. Building a platform is, in a way, like raising a child: you can try to steer them in a certain direction, but it's important to leave them space to breathe and find their own path.
+
+### **Will there be Complex Content Filters and All That Jazz?**
```
Comment (essential-randomness) [inline docs/users/welcome.md:48]: Mostly same as above. The way I'd change is is I'd keep the content as you've written it out but structure it in a way that makes it easier to see how things work at a glance:
@: does this
+: does that

File: docs/users/welcome.md (hunk 1 above)
Comment (QuestionableTangerine) [reply docs/users/welcome.md:40]: Post deletions & necroposting were both questions users have asked about in terms of boba etiquette & that didn't quite seem appropriate for the 'main' etiquette section. I was also planning on putting some sort of 'how do i know what board to post on'? section in here since that was one of the more popular requests on this post: https://v0.boba.social/!bobaland/thread/45d78a36-f465-428e-a1d4-c1fdae6e9e52

The 'is bobaboard a [x] clone? is more general though and i agree that could prob be cut/moved somewhere else

Comment (QuestionableTangerine) [reply docs/users/welcome.md:40]: Actually correction: the "can i necropost" is currently unchanged from the orig bobadoc -- am not sure why that section got clipped off when it made the jump to github but if that was intentional i don't mind moving the FAQ?

**Comment by enigmalea** [comment]
This document has been renamed and updated several times since this PR has been submitted.
